class InstantiateCSVError(Exception):
    """ класс-исключение для проверки повреждения файла CSV"""
    pass


